packages/server/src/websocket/dto/messagerepository-websocket.dto.ts

```diff
@@ -25,7 +25,11 @@ export class TakeFromQueueDto {
 
   @IsOptional()
   @IsInt()
-  limit: number
+  limit?: number
+
+  @IsOptional()
+  @IsInt()
+  limitSize?: number
```

Comment:
```suggestion
  limitBytes?: number
```

To align with `total_bytes` in the Pickup V2 spec (and also be more explicit in the units)